controller/array_action/array_mediator_svc.py

```diff
@@ -840,3 +845,14 @@ def get_array_fc_wwns(self, host_name):
             logger.error(msg="Failed to get array fc wwn. Reason "
                              "is: {0}".format(ex))
             raise ex
+
+    def _get_host_by_name(self, host_name):
+        filter_value = 'name={}'.format(host_name)
+        hosts_list = self.client.svcinfo.lshost(filtervalue=filter_value).as_list
+        if not hosts_list:
+            raise array_errors.HostNotFoundError(host_name)
+        return hosts_list[0]
```

Comment:
shouldn't it be `as_single_element` instead of `as_list` + `[0]`?

Reply:
done

Reply:
now it's no longer a host *list*

Reply:
done